suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { IconButton, Box, Text } from '@suite-native/atoms';
+
+type AccountDetailScreenHeaderProps = {
+    accountName?: string;
+};
+
+export const AccountDetailScreenHeader = ({ accountName }: AccountDetailScreenHeaderProps) => {
+    const navigation = useNavigation();
+    return (
+        <Box
+            flexDirection="row"
+            justifyContent="space-between"
+            alignItems="center"
+            paddingHorizontal="large"
+        >
+            <IconButton
+                colorScheme="gray"
+                isRounded
+                size="large"
+                iconName="chevronLeft"
+                onPress={() => navigation.goBack()}
+            />
+            <Text>{accountName}</Text>
+            <IconButton
+                colorScheme="gray"
+                isRounded
+                size="large"
+                iconName="settings"
+                onPress={() => {}}
```

Comment:
should we have this here if it is not working? I would vote for removing it until it will be used, otherwise it could be quite confusing for users. Maybe @mnuky should decide?

Reply:
is it the settings button on top right? If yes, remove it, until we do the Asset settings feature.

Reply:
Daaamn. I think it will be easier to just create the screen with some super simple option maybe at least to delete the xpub. Because if I remove it, I will have to restyle the header and then restyle it again when we create the screen 😅😭

Reply:
We should have some kind of header component that would break always when we add or remove icon in first place, because we have this problem nearly everywhere.

Anyway for now, I will be okay with something like `opacity: 0` with explanantion comment why it is here.

Reply:
It's still not 100% finished screen, but I feel like adding this functionality will help also QA with testing and it's been annoying me for some time that I have to delete all accounts if I want to delete just one. So here it is. Button color etc is being discussed in Figma with UX.

https://user-images.githubusercontent.com/36101761/200740339-085f4549-e14c-46bd-b467-0ad93034cc46.mov